dev-requirements.txt

```diff
@@ -8,5 +8,6 @@ pre-commit == 2.20.0
 pytest == 7.2.0
 pytest-cov == 4.0.0
 pytest-mock == 3.14.0
+pytest_benchmark == 4.0.0
```

Comment:
Why is it needed now?

Reply:
My local environment wouldn't work without it, so I added it